app/main/views/send.py

```diff
@@ -517,19 +516,27 @@ def _check_messages(service_id, template_id, upload_id, preview_row):
         current_service,
         show_recipient=False,
     )
+
+    allow_list = []
+    if current_service.trial_mode:
+        # Adding the simulated numbers to allow list
+        # so they can be sent in trial mode
+        for user in Users(service_id):
+            allow_list.extend([user.name, user.mobile_number, user.email_address])
+        # Failed sms number
+        allow_list.extend(["simulated user", "+14254147167", "simulated@simulated.gov"])
+        # Success sms number
+        allow_list.extend(
+            ["simulated user two", "+14254147755", "simulatedtwo@simulated.gov"]
```

Comment:
Would it make sense to have the username reflect which succeeds and which fails?

```suggestion
        # Failed sms number
        allow_list.extend(["simulated user (fail)", "+14254147167", "simulated@simulated.gov"])
        # Success sms number
        allow_list.extend(
            ["simulated user (success)", "+14254147755", "simulatedtwo@simulated.gov"]
```